requirements/scenarios.md

```diff
@@ -214,7 +214,7 @@ They pushed unsigned content to existing tags.
    The scenario simply calls out a compromise that must be discoverable through forensic logging of who, when and what was pushed.
    The logged updates must include previous digests that represented updated tags allowing a registry, or external tools, to reset its original state.
 1. A registry may choose to make repos, or the entire registry, restricted to pushing only signed content, and potentially only signed content with one or more keys.
-   Notary v2 doesn't require this capability, rather highlights the scenario for registry operators to innovate means to secure from this scenario.
+   Notary Project doesn't require this capability, rather highlights the scenario for registry operators to innovate means to secure from this scenario.
```

Comment:
Is it the _Notary Project_ that doesn't require the capability or the specification?

Reply:
Thanks. Update it to `The Notary Project`. cc @toddysm for comments.

Reply:
@mattfarina This sentence is under `Scenario #7: Repository Compromise - Non Signed Content`. It depends on the implementation or registries for security, so I rephased the sentence to below:

```
A registry may choose to restrict repositories or the entire registry to pushing only signed content, and 
potentially only signed content with one or more keys. With this, attackers cannot push unsigned content to 
the registry, mitigating the threat described by scenario #7
```